include/fe/fe_base.h

```diff
@@ -795,7 +795,7 @@ void FEGenericBase<OutputType>::compute_dual_shape_functions ()
 }
 
 template <typename OutputType>
-void FEGenericBase<OutputType>::compute_dual_shape_coeffs(const std::vector<Real> & /*JxW*/, std::vector<std::vector<OutputShape>> & /*phi_vals*/)
+void FEGenericBase<OutputType>::compute_dual_shape_coeffs(const std::vector<Real> & /*JxW*/, const std::vector<std::vector<OutputShape>> & /*phi_vals*/)
```

Comment:
As long as there are more changes to make anyway, let's just turn this commit into a fixup on top of the compilation-breaking commit it fixed?